build.gradle

```diff
@@ -42,18 +42,22 @@ task coverage(type: JacocoReport) {
 
 dependencies {
     String jUnitVersion = '5.4.0'
-    String javaFxVersion = '17.0.7'
+    String javaFxVersion = '17.0.11'
 
     implementation group: 'org.openjfx', name: 'javafx-base', version: javaFxVersion, classifier: 'win'
+    implementation group: 'org.openjfx', name: 'javafx-base', version: javaFxVersion, classifier: 'mac-aarch64'
```

Comment:
@ryanozx why do we need to add these implementation lines? ab3 was working for me without adding these  lines but changing the javafx version etc

Reply:
I was going to close the pull request but to answer your question:
- Gradle downloads the Intel version of JavaFX for the Mac by default - maybe there's a different way to set it up, but I couldn't execute `./gradlew test` or `./gradlew run` successfully without these lines. I should also note that I was trying to get it to work with OpenJDK 17 rather than using the Azul Zulu build, the former of which does not come bundled with JavaFX.

Reply:
Resolved as JDK distribution changed to Azul 17